ydb/docs/en/core/reference/docker/install.md

```diff
@@ -0,0 +1,11 @@
+# Installing Docker to work with {{ ydb-short-name }}
+
+Before using the Docker image {{ ydb-short-name }}, you need to install and configure the Docker environment. To do this, you can use the official documentation:
+
+- [Linux](https://docs.docker.com/desktop/install/linux-install/)
+- [macOS](https://docs.docker.com/desktop/install/mac-install/)
+- [Windows](https://docs.docker.com/desktop/install/windows-install/)
+
+Alternative installation methods like [colima](https://github.com/abiosoft/colima) are also acceptable.
+
+The installation of {{ ydb-short-name }} itself occurs at the [first launch](start.md).
```

Comment:
This article needs the same story about Rosetta on Apple Silicon as the quickstart currently has. Let's move the existing text to an `_include` file and `{% include ... %}` it in both places.